frontend/src/main/pages/HomePage.js

```diff
@@ -35,6 +38,9 @@ export default function HomePage() {
                 <h1 data-testid="homePage-title" style={{ fontSize: "75px", borderRadius: "7px", backgroundColor: "white", opacity: ".9" }} className="text-center border-0 my-3">
                     {greeting}, {currentUser.root.user.githubLogin}
                 </h1>
+                <h2 data-testid="homePage-info">
+                     This app is intended as a replacement for the <a href="https://ucsb-cs-github-linker.herokuapp.com">ucsb-cs-github-linker</a> app used in many courses at UCSB, as well as some courses at other universities. 
+                </h2>
             </BasicLayout>
         </div>
     );
```

Comment:
Ok, you're probably gonna hate me for this but... I can't let this stand.

What on earth is going on on this page?  Why do we have a hard coded section for cgaucho, and a complete repeat of all of the same code?

This makes no sense at all.

At most, we should be putting the conditional `if (!currentUser || !currentUser.loggedIn) {` around 
```
greeting}, {currentUser.root.user.githubLogin}
```

This is ... not good code.  And even though you didn't write it, and it's not your fault, as soon as you touch the file, you own it.

So, can you try to figure out what's going on here, and clean this up a bit?

Reply:
Hi,


 I was looking at the code and it seems like that piece of code is there so that it can test if the greeting shows up or not because on the tests, there is no  GitHub username to pull from. Should I delete that section still?

Reply:
you dont really need the cgaucho part.
even if you delete that part the tests still check for the variable github id.

So now you don't only remove the that part from the HomePage.js you also make sure the tests it was testing for are also not there.

Reply:
```
                <h1 data-testid="homePage-title" style={{ fontSize: "75px", borderRadius: "7px", backgroundColor: "white", opacity: ".9" }} className="text-center border-0 my-3">
                    {greeting}, {currentUser.root.user.githubLogin}
```
                    
                    
This line of code is being tested regardless of if you remove the if user is not logged in part.